rdf2vec/walkers/anonymous.py

```diff
@@ -4,10 +4,35 @@
 from hashlib import md5
 
 class AnonymousWalker(RandomWalker):
+    """Defines the anonymous walker of the walking strategy.
```

Comment:
Each walker is actually a walking strategy. So I would just write "Defines the anonymous walking strategy."